tunnel-server/index.ts

```diff
@@ -69,7 +69,9 @@ const sshServer = createSshServer({
   },
   onHello: (clientId, tunnels) => JSON.stringify({
     clientId,
-    baseUrl: BASE_URL.toString(),
+    // TODO: backwards compat, remove when we drop support for CLI v0.0.34
+    baseUrl: { hostname: BASE_URL.hostname, port: BASE_URL.port, protocol: BASE_URL.protocol },
```

Comment:
LGTM :)
Basically 0.0.35 still use the old baseURL I think.

I think we can create the change of the CLI in another PR and push release afterward. (0.0.360 (in the meantime, return the original baseUrl functionility).

There's no need to merge code with branching on baseUrl/RootUrl as the server should support both cases after deployment (and before, it just support v0.0.35)